shanoir-ng-studies/src/main/java/org/shanoir/ng/study/service/RabbitMQStudiesService.java

```diff
@@ -127,7 +127,7 @@ public void challengeSubscription(final String studyStr) {
 			Long userId = event.getUserId();
 			Long studyId = Long.valueOf(event.getObjectId());
 			// Get the study
-			Study studyToUpdate = studyRepo.findOne(studyId);
+			Study studyToUpdate = studyRepo.findById(studyId).orElseThrow();
```

Comment:
Throw something ? Or null ?

Reply:
.orElseThrow() throws already a NoSuchElementException by default, the nice thing of this method, if I am right